### ISSUE TITLE
in the method paranteshes

### ISSUE THREAD

**Issue by maliekici** [Dugahar/Java-odev-2.2#1]
utku **wtf** bro. this are is a github we use english language this area whatever. If I come to your code I went to this part DortIslem.java You have specified the values ​​you will assign as int inside the parentheses as 'int'. Do you have to do this? or java is silly technology ?

**Comment by Dugahar** [comment]
Yes you have to specify the type of the value inside the parenthesis. For the second question, there is a saying I saw on the internet : Life is to short to learn java.

**Comment by Dugahar** [comment]
Yes you have to specify the type of the value inside the parenthesis. For the second question, there is a saying I saw on the internet : Life is to short to learn java.
